### ISSUE TITLE
Base mission functionality, stage 1

### ISSUE THREAD

**Issue by Bubbus** [Bubbus/CA-Halloween-2021#5]
Make the following available in the `base` branch, and then merge into the mission branches:

- [x] Latest framework features (including from `zen-refactor`).
- [x] Imported anomaly scripts from CAyZ, Op Glittering and Starma.
- [x] Init-field-compatible ways to initialise an anomaly, so that they can be pre-placed or created via trigger.
- [x] Merge into mission branches.

**Comment by Bubbus** [comment]
Blocked on #4

**Comment by dgibso29** [comment]
Block can be removed pending your approval of #4

**Comment by Bubbus** [comment]
Done, mission branches ready for dev work.
